### PR TITLE
Respond issues

### PR DESCRIPTION
**Pull request by minju25kim**
- fixed several issues
- form validation for date
- populate db's .env file path
- error-handling for promises
- remove some aria label

### CODE REVIEW

**Review by AsmahanM — APPROVED** [review]
Great work!

File: routes/add-pet.js [hunk 1]
```diff
@@ -67,7 +75,7 @@ function get(request, response) {
     response.send(html);
   } catch (error) {
     console.error(error);
-    response.status(404).send("<h1>Error handling</h1>");
+    response.status(500).send("<h1>Error handling</h1>");
```
Comment (AsmahanM) [inline routes/add-pet.js:78]: Good reminder to be conscious of the status code for next time.

File: routes/birthdays.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ function calcAge(data) {
 function get(request, response) {
   const filterTerm = request.query.type;
   let filterType = "";
-  if (filterTerm & (filterTerm != 0)) {
+  if (filterTerm && (filterTerm != 0)) {
```
Comment (AsmahanM) [inline routes/birthdays.js:26]: This was a good catch

**Review by Glombort — APPROVED** [review]
Nice, everything looks great:)

**Pull request merged**